crates/red_knot/src/args.rs

```diff
@@ -247,3 +251,17 @@ impl From<OutputFormat> for ruff_db::diagnostic::DiagnosticFormat {
         }
     }
 }
+
+/// Control when colored output is used.
+#[derive(Copy, Clone, Hash, Debug, PartialEq, Eq, PartialOrd, Ord, Default, clap::ValueEnum)]
+pub(crate) enum TerminalColor {
+    /// Automatically detect if color support is available on the terminal.
+    #[default]
+    Auto,
+
+    /// Always display colors
```

Comment:
Minor: use a full stop here as well for consistency?
```suggestion
    /// Always display colors.
```